test/test-tool-model.js

```diff
@@ -1,36 +0,0 @@
-const { assert } = require('chai');
-const { mock_api } = require('./conftest');
-const { api } = require('../app/api_clients/kubernetes.js');
-const { Deployment, ModelSet, Pod, Tool } = require('../app/models');
-
-
-describe('Tool model', () => {
```

Comment:
Are we happy to get rid of this test?

Reply:
I moved this into test/models/kubernetes.js along with some small changes